docs/sphinx/source/installation.rst

```diff
@@ -34,49 +33,153 @@ There are many ways to install Python on your system, but the Anaconda
+
+#. **Download** the `GitHub Desktop <https://desktop.github.com>`_ application.
+#. **Fork** the pvlib-python project by clicking on the "Fork" button on
+   the upper right corner of the
+   `pvlib-python GitHub page <https://github.com/pvlib/pvlib-python>`_.
+#. **Clone** your fork to your computer using the GitHub Desktop application
+   by clicking on the *Clone to Desktop* button on your fork's homepage.
+   This button is circled in the image below.
+
+.. image:: _images/clonebutton.png
 
-If you have trouble, see the `Anaconda
-FAQ <http://docs.continuum.io/anaconda/faq.html>`_, Google your error
-messages, or make a new issue on our `Issues
-page <https://github.com/pvlib/pvlib-python/issues>`_.
+Please see GitHub's
+`Forking Projects <https://guides.github.com/activities/forking/>`_ and
+`Fork A Repo <https://help.github.com/articles/fork-a-repo/>`_ for
+more details.
 
+.. _setupenvironment:
 
-Working at the bleeding edge
-----------------------------
+Set up a virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 We strongly recommend working in a `virtual environment
 <http://astropy.readthedocs.org/en/latest/development/workflow/
-virtual_pythons.html>`_ if you're going to use the development versions
-of the code. There are many ways to use virtual environments in Python,
-but Anaconda again provides the easiest solution:
+virtual_pythons.html>`_ if you're going to use an editable version
+of the library. You can skip this step if:
 
-#. Create a new conda environment for pvlib and pre-install a
-   handful of packages into the environment:
-   ``conda create --name pvlibdev python pandas scipy``
-#. Activate the new environment: ``source activate pvlibdev``
-#. Install the latest development version of pvlib:
+#. You already have Anaconda or another scientific Python distribution
+#. You don't mind polluting your Python installation with your
+   development version of pvlib.
+#. You don't want to work with multiple versions of pvlib.
 
-  #. If you don't plan to modify the source-code:
-     ``pip install git+https://github.com/pvlib/pvlib-python.git``
-  #. If you do plan to modify the source code:
-     Use the GitHub GUI application or git command-line tool to
-     clone this repository to your computer, then navigate your
-     command-line to the top-level pvlib-python directory,
-     then ``pip install -e .``
+There are many ways to use virtual environments in Python,
+but Anaconda again provides the easiest solution. These are often
+referred to as *conda environments*, but they're the same for our purposes.
 
-#. You may also consider installing additional packages into your
-   development environment:
-   ``conda install jupyter ipython seaborn nose flake8``
+#. **Create** a new conda environment for pvlib and pre-install
+   the required packages into the environment:
+   ``conda create --name pvlibdev python pandas scipy``
+#. **Activate** the new conda environment: ``source activate pvlibdev``
```

Comment:
This is even more understandable than upsteam!

I never understood from [[http://conda.pydata.org/docs/using/envs.html#create-an-environment|Create an environment]]
`conda create --name snowflakes biopython`
What? Nicely written.